packages/rs-drive-abci/src/execution/platform_events/state_transition_processing/execute_event/v0/mod.rs

```diff
@@ -73,6 +80,24 @@ where
                         )
                         .map_err(Error::Drive)?;
 
+                    // Apply signature verification costs
+                    if let Some(signature_verifications) = signature_verifications {
+                        let total_verification_cost: Credits =
+                            signature_verifications.iter().fold(0, |acc, op| {
```

Comment:
you need to handle errors, I don't think fold is right.

Reply:
Fixed